_posts/2017-05-02-textworkshop17.md

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
Thank you @kbenoit. This post is great. Captures intentions, richness of specific work done, as well as the spirit of rOpenSci community! I have only minor suggested edits and @sckott agrees we can post as soon as you incorporate these and give us the ok.